### PR TITLE
Add UsersList container, test

### LINKED ISSUE #5: Add container for users
container = class component (e.g. usersList.js)

### PR DESCRIPTION
**Pull request by mattwr18**
fixes #5

### CODE REVIEW

**Review by joaopapereira — APPROVED** [review]
Looks good.
Going to merge

File: webpack.config.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ const htmlPlugin = new HtmlWebPackPlugin({
 });
 
 module.exports = {
+  devServer: { historyApiFallback: true },
```
Comment (Kachulio1) [inline webpack.config.js:11]: Why do we have this here

**Review by Kachulio1 — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (mattwr18) [reply webpack.config.js:11]: it was suggested as a fix to `Content Security Policy: The page's settings blocked the loading of a resource at eval ("default-src")`

see https://stackoverflow.com/questions/46422290/webpack-dev-server-react-content-security-policy-error?answertab=active#tab-top

File: src/containers/UsersList.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React, { Component } from "react";
+import { Header } from "semantic-ui-react";
+
+export class UsersList extends Component {
+  render() {
+    return <Header>Volunteers Directory</Header>;
+  }
+}
```
Comment (Kachulio1) [inline src/containers/UsersList.js]: new line

File: webpack.config.js (hunk 1 above)
Comment (Kachulio1) [reply webpack.config.js:11]: ok

File: src/fixtures/users.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    timezone_offset: -18000,
+    country_code: "BR",
+    status_count: 0,
+    deleted_at: null,
+    event_participation_count: 0,
+    can_see_dashboard: false,
+    skill_list: [ ],
+    title_list: [ ]
+    },
+]
```
Comment (Kachulio1) [inline src/fixtures/users.js]: new line

Comment (mattwr18) [reply src/fixtures/users.js]: add a new line? 
will we set up linting?

File: src/tests/UserList.test.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          })
+        }
+      />
+    </StaticRouter>
+  );
+
+  it("should have a header Volunteers Directory", () => {
+    expect(wrapper.find("Header").text()).toBe("Volunteers Directory");
+  });
+});
```
Comment (Kachulio1) [inline src/tests/UserList.test.js]: new line

File: src/fixtures/users.js (hunk 3 above)
Comment (Kachulio1) [reply src/fixtures/users.js]: yes

**Review by Kachulio1 — APPROVED** [review]

**Pull request merged**